libcudacxx/include/cuda/std/detail/libcxx/include/__cuda/barrier.h

```diff
@@ -27,6 +27,7 @@ _CCCL_IMPLICIT_SYSTEM_HEADER
 
 #include "../cstdlib"                // _LIBCUDACXX_UNREACHABLE
 #include "../__type_traits/void_t.h" // _CUDA_VSTD::__void_t
+#include <cuda/ptx>                  // cuda::ptx::*
```

Comment:
This should now be
```suggestion
#include "../__cuda/ptx.h"                  // cuda::ptx::*
```

Reply:
Good catch! I hadn't considered this include.